# HG changeset patch
# User Rik <rik@octave.org>
# Date 1382210371 25200
#      Sat Oct 19 12:19:31 2013 -0700
# Node ID 10d3bd4ec98148b6f14ec7cddddc8af3d820c721
# Parent  dba2e06dcdb5b92621b35c88c488eed757a8c32c
maint: Move datetick from time/ to plot/appearance/ directory

* scripts/plot/appearance/datetick.m: Moved from time/ dir.

* scripts/plot/appearance/module.mk: Added datetick to build system.

* scripts/time/module.mk: Removed datetick from time/ dir build.

diff --git a/scripts/time/datetick.m b/scripts/plot/appearance/datetick.m
rename from scripts/time/datetick.m
rename to scripts/plot/appearance/datetick.m
diff --git a/scripts/plot/appearance/module.mk b/scripts/plot/appearance/module.mk
--- a/scripts/plot/appearance/module.mk
+++ b/scripts/plot/appearance/module.mk
@@ -7,16 +7,17 @@ plot_appearance_PRIVATE_FCN_FILES = \
 plot_appearance_FCN_FILES = \
   plot/appearance/__clabel__.m \
   plot/appearance/__getlegenddata__.m \
   plot/appearance/axis.m \
   plot/appearance/box.m \
   plot/appearance/caxis.m \
   plot/appearance/clabel.m \
   plot/appearance/daspect.m \
+  plot/appearance/datetick.m \
   plot/appearance/diffuse.m \
   plot/appearance/grid.m \
   plot/appearance/gtext.m \
   plot/appearance/hidden.m \
   plot/appearance/legend.m \
   plot/appearance/orient.m \
   plot/appearance/pbaspect.m \
   plot/appearance/shading.m \
diff --git a/scripts/time/module.mk b/scripts/time/module.mk
--- a/scripts/time/module.mk
+++ b/scripts/time/module.mk
@@ -4,17 +4,16 @@ time_FCN_FILES = \
   time/addtodate.m \
   time/asctime.m \
   time/calendar.m \
   time/clock.m \
   time/ctime.m \
   time/date.m \
   time/datenum.m \
   time/datestr.m \
-  time/datetick.m \
   time/datevec.m \
   time/eomday.m \
   time/etime.m \
   time/is_leap_year.m \
   time/now.m \
   time/weekday.m
 
 FCN_FILES += $(time_FCN_FILES)
